adapters/parent/pom.xml

```diff
@@ -132,8 +132,12 @@
                         <!-- use Base64 encoder/decoder that is compatible with vert.x 3 -->
                         <JAVA_OPTIONS>
                           -Dvertx.json.base64=legacy
-                          -Djava.util.logging.manager=org.jboss.logmanager.LogManager</JAVA_OPTIONS>
+                          -Djava.util.logging.manager=org.jboss.logmanager.LogManager
+                        </JAVA_OPTIONS>
                       </env>
+                      <runCmds>
+                        <run>chmod 755 /opt/hono/run-java.sh</run>
```

Comment:
`<fileMode>755</fileMode>` below can then be removed.

(And typo in commit message: "shall".)